libxrdp/xrdp_sec.c

```diff
@@ -2486,8 +2546,15 @@ xrdp_sec_process_mcs_data(struct xrdp_sec *self)
                     return 1;
                 }
                 break;
+            case SEC_TAG_CLI_MONITOR_EX:  /* CS_MONITOR_EX     0xC008 */
+                if (xrdp_sec_process_mcs_data_monitors_ex(self, s) != 0)
+                {
+                    LOG(LOG_LEVEL_ERROR,
+                        "Processing [MS-RDPBCGR] TS_UD_CS_MONITOR_EX failed");
+                    return 1;
+                }
+                break;
             /* CS_MCS_MSGCHANNEL 0xC006
-               CS_MONITOR_EX     0xC008
```

Comment:
Why remove this?

Reply:
Because we're now processing CS_MONITOR_EX under the `case SEC_TAG_CLI_MONITOR_EX:` tag, so it no longer makes sense to have this in the comment.